appjs/views/edit_project.js

```diff
@@ -466,7 +547,7 @@ var EditProject = BaseView.extend(require('./mixins/actions'), require('./mixins
         return _.contains(this.model.getConfig().themes, t.get('slug'));
       }, this)) : this.app.themes.models;
     availableThemes = availableThemes || this.app.themes.where({slug : 'generic'});
-    twitterHandles = _.object(this.app.themes.pluck('slug'), this.app.themes.pluck('twitter_handle'));
+    this.twitterHandles = _.object(this.app.themes.pluck('slug'), this.app.themes.pluck('twitter_handle'));
```

Comment:
Saved variable as part of `this` so that it can be accessed from `getTwitterCount` (which runs on `keyup` from `#shareText`) and not generated every time.